packages/first-boot-wizard/tests/test_firstbootwizard.lua

```diff
@@ -7,6 +7,7 @@ local fs = require("nixio.fs")
 local fbw_utils = require('firstbootwizard.utils')
 
 local uci = nil
+config.log = function () end
```

Comment:
what is the purpose of this one? doesn't seem to be used anywhere, nor adding any useful functionality

Reply:
This is to silent some logging performed by different modules during the tests. We should have a config.disable_logging() or something link that.

Reply:
sounds good ... sorry that I can't help with that :(

Reply:
Thanks for reviewing! it is very helpful